src/components/Chart/CrosshairTooltip/index.jsx

```diff
@@ -1,17 +1,22 @@
 import dayjs from 'dayjs';
 import PropTypes from 'prop-types';
+import { useMedia } from 'react-use';
 
 import { Typography } from '../../../Theme';
-import { formatChartValueByType } from '../../../utils';
+import { formatChartValueByType, getWeekFormattedDate } from '../../../utils';
 import { Wrapper } from './styled';
 
-const CrosshairTooltip = ({ title, active, payload, dataType }) => {
+const CrosshairTooltip = ({ title, active, isWeeklyActive, payload, dataType }) => {
+  const below500 = useMedia('(max-width: 500px)');
+
   if (active && payload && payload.length) {
     const { time, value } = payload[0].payload;
 
     return (
       <Wrapper>
-        <Typography.Text color={'text10'}>{dayjs(time).format('MMMM D, YYYY')}</Typography.Text>
+        <Typography.Text color={'text10'}>
+          {isWeeklyActive ? getWeekFormattedDate(time, below500) : dayjs(time).format('MMMM D, YYYY')}
```

Comment:
might be interesting to move this ternary to a function.

Reply:
I'll move this in a `formatChartDate` util.